src/components/page/Loading/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import './style.css';
+
+/** interface for Ripple props */
+export interface RippleProps {
+  borderColor?: string;
+  borderStyle?: string;
+  borderWidth?: string;
+  height?: string;
+  minHeight?: string;
+  width?: string;
```

Comment:
If we have these all defined in default props then they should not be optional, no?